cmd/maestroctl/cmd/plan/planStatus.go

```diff
@@ -0,0 +1,167 @@
+package plan
+
+import (
+	"encoding/json"
+	"fmt"
+	maestrov1alpha1 "github.com/maestrosdk/maestro/pkg/apis/maestro/v1alpha1"
+	"github.com/spf13/cobra"
+	"github.com/xlab/treeprint"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+	"log"
+)
+
+func NewPlanStatusCmd() *cobra.Command {
+	statusCmd := &cobra.Command{
+		Use:   "status",
+		Short: "Shows the status of all plans to an particular instance.",
+		Long: `
+	# View plan status
+	maestroctl plan status --instance=<instanceName> --kubeconfig=<$HOME/.kube/config>`,
+		Run: planStatusCmd,
+	}
+
+	statusCmd.Flags().StringVar(&instance, "instance", "", "The instance name available from 'kubectl get instances'")
+	statusCmd.Flags().StringVar(&kubeConfig, "kubeconfig", "", "The file path to kubernetes configuration file; defaults to $HOME/.kube/config")
+	statusCmd.Flags().StringVar(&namespace, "namespace", "default", "The namespace where the instance is running.")
+
+	return statusCmd
+}
+
+func planStatusCmd(cmd *cobra.Command, args []string) {
+
+	instanceFlag, err := cmd.Flags().GetString("instance")
+	if err != nil || instanceFlag == "" {
+		log.Printf("Flag Error: %v", err)
+	}
```

Comment:
You should `os.Exit(1)` here.  Not sure what the right error code should be, but exit.

Reply:
`os.Exit(1)` seems as the appropriate error code. I will address it in my next commit.